doc/yast-storage-requirements.md

```diff
@@ -0,0 +1,346 @@
+# YaST Storage Features and Requirements
+
+This is about *libstorage* and the *yast-storage* Ruby code.
+
+##AIs summary
+
+* AI Lukas: ML: libstorage-ng@opensuse.org
+* AI PM: Clarify requirements on FAT partitions
+* AI lmb, Holgi: Claryfy requirements on filesystems (see below)
+* AI Arvin: clarify why: reate partitions on RAID (if there already is a partition table; cannot create a partition table if there is not one already
+* AI Architecture maintainers: Clarify which configurations of RAID are bootable, possibly under which conditions
+* AI: Veryfy support of resizing logical volumes for LVM
+* AI: Clarify why we do not support encryption of root filesystem, which tools would possibly need to be fixed
+* AI: speak with different parties which requirements they have on automated proposal (Cloud, SES, SAP, ...)
+* AI: Define when dedicated boot partition is needed before merging into the 
+* AI: Find otu if EVMS support for upgrade is necessary (when was it last supported?)
+* AI: Provide specifics for zFCP devices
+* AI: Define requirements on /boot/zipl partition
+* AI: Provide details about specific tools for first sector writing, reasoning for two copies of kernel (why two?) for DASD
+
+
+## Disks and Partitions Simple case:
+
+   * Disk  
+       * Partition table
+           * Partition  
+               * File system  
+                      * File
+                      * File
+                      * File
+                      * ...
+           * Partition  
+               * File system  
+                      * File
+                      * ...
+           * Partition  
+               * ...
+           * ...
+
+## Advanced Cases
+
+This covers various ways to combine disks and/or use advanced
+ways to manage the disk space provided by the disks
+
+###RAID (Redundant Array of Independent Disks)
+
+   * Hardware RAID: mostly automatic and transparent
+       * Behaves like any regular disk
+       * No need for any special handling (or even detection)
+   * Linux Software RAID / BIOS RAID (type dependent on BIOS):  
+       * mdadm RAID: https://en.wikipedia.org/wiki/Mdadm
+       * dmraid (Device Mapper RAID): https://en.wikipedia.org/wiki/Device_mapper ?
+
+BIOS-managed raid is configured in BIOS (and Linux system detects it), Linux softwre RAID must be configured in Linux
+
+RAID levels:
+
+   * **RAID 0**: striping; performance improvement due to parallel disk I/O; less fault tolerance than one disk
+   * **RAID 1**: mirroring; improved fault tolerance, same performance as one disk
```

Comment:
RAID 1 can have better performance for reading, see https://en.wikipedia.org/wiki/RAID.